### ISSUE TITLE
Login.aspx

### ISSUE THREAD

**Issue by pallavikotturu** [pallavikotturu/HW8#7]
<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="Login.aspx.cs" Inherits="Kotturu2.Login" %>

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head runat="server">
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Home</title>
</head>

<body>
    <form id="form1" runat="server">
    
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="container">
            <div class="navbar-header">
                <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="brand" href="#">
                    <img src="Logo.PNG" class="ui-icon-image" height="50" width="50" alt="" />
                </a>
            </div>

            <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
                <ul class="nav navbar-nav">
                    <li><a href="Home.aspx">Home</a></li>
                    <li class="active"><a href="Login.aspx">Login</a></li>
                    <li><a href="AboutYou.aspx">About You</a></li>
                </ul>
            </div>

        </div>
    </div>

         <div class="container" style="margin-top: 5%">
        <div class="col-md-4 col-md-offset-4">
            <div class="panel panel-default">
                <div class="panel-heading"><h1 class="panel-title"><strong>Log In </strong></h1></div>
                <div class="panel-body">
                    <form role="form">
                        <div class="form-group">
                            <div class="col-md-12">
                            <label for="exampleInputEmail1">Username</label>
                         <asp:TextBox runat="server" class="txt"  id="txtAccessNet" name="txtAccessNet" Text="" OnClick="btnLogin_Click" ></asp:TextBox>
                        </div>
                            </div>
                        <div class="form-group">
                            <div class="col-md-12">
                            <label for="exampleInputPassword1">Password</label>
                         <asp:TextBox runat="server" class="txt"  id="txtPassword" name="txtPassword" Text="" OnClick="btnLogin_Click" ></asp:TextBox>
                        </div>
                            </div>
                        <br />
                        <div class="form-group">
                            <div class="col-md-12">
                        <asp:Button runat="server" class="btn btn-md btn-success"  id="btnLogin" name="btnLogin" Text="Login" OnClick="btnLogin_Click" ></asp:Button>
                        </div>
                            </div>
                        <br />
                        <br />
                      <div class="form-group">
                    <div class="col-md-12">
                        <br />
                        <asp:Label id="lblError" name="lblError" runat="server" style="position: relative" Text="Your Username and Password are required fields."></asp:Label>
                    </div>
                </div>

                    </form>
                </div>
            </div>
        </div>

             </form>
</body>
</html>

**Comment by pallavikotturu** [comment]
<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="Login.aspx.cs" Inherits="Kotturu2.Login" %>

<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head runat="server">
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Home</title>
</head>

<body>
    <form id="form1" runat="server">

```
<div class="navbar navbar-inverse navbar-fixed-top">
    <div class="container">
        <div class="navbar-header">
            <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="brand" href="#">
                <img src="Logo.PNG" class="ui-icon-image" height="50" width="50" alt="" />
            </a>
        </div>

        <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
            <ul class="nav navbar-nav">
                <li><a href="Home.aspx">Home</a></li>
                <li class="active"><a href="Login.aspx">Login</a></li>
                <li><a href="AboutYou.aspx">About You</a></li>
            </ul>
        </div>

    </div>
</div>

     <div class="container" style="margin-top: 5%">
    <div class="col-md-4 col-md-offset-4">
        <div class="panel panel-default">
            <div class="panel-heading"><h1 class="panel-title"><strong>Log In </strong></h1></div>
            <div class="panel-body">
                </div>
                    <div class="form-group">
                        <div class="col-md-12">
                        <label for="exampleInputEmail1">Username</label>
                     <asp:TextBox runat="server" class="txt"  id="txtAccessNet" name="txtAccessNet" Text="" OnClick="btnLogin_Click" ></asp:TextBox>
                    </div>
                        </div>
                    <div class="form-group">
                        <div class="col-md-12">
                        <label for="exampleInputPassword1">Password</label>
                     <asp:TextBox runat="server" class="txt"  id="txtPassword" name="txtPassword" Text="" OnClick="btnLogin_Click" ></asp:TextBox>
                    </div>
                        </div>
                    <br />
                    <div style="margin-left: 80px">
                        <asp:Button runat="server" class="btn btn-md btn-success"  id="btnLogin" name="btnLogin" Text="Login" OnClick="btnLogin_Click" ></asp:Button>
    </div>
                    <div class="form-group">
                        <div class="col-md-12">
                    </div>
                        </div>
                    <asp:Label id="lblError" name="lblError" runat="server" style="position: relative" Text="* Your Username and Password are required fields!"></asp:Label>
                    <br />
                    <br />
                  <div class="form-group">
                <div class="col-md-12">
                    <br />
                </div>
            </div>

                    <div class="col-page col-sm-12 col-md-12">
            <div class="img-publicacion-home">
                <p>
                    <br />
                    <br />
                    <hr />
                    Footer:  
<a href="https://www.linkedin.com/in/pallavikotturu"><span class="label label-info">Author</span></a> 
<a href="https://cis.temple.edu/"><span class="label label-info">Temple University CIS</span></a> 
| <i class="icon-calendar"></i> November 8th, 2016 at 10:15 am
| <i class="icon-comment"></i> <a href="#">3 Comments</a>
| <i class="icon-share"></i> <a href="#">39 Shares</a>
```

</p>

```
                </div>
              </div>
                </form>
            </div>
        </div>
    </div>

         </form>
```

</body>
</html>

**Comment by pallavikotturu** [comment]
`

<%@ Page Language="C#" AutoEventWireup="true" CodeBehind="Login.aspx.cs" Inherits="Kotturu2.Login" %>



<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head runat="server">
    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
    <title>Home</title>
</head>

<body>
    <form id="form1" runat="server">
    
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="container">
            <div class="navbar-header">
                <button button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="brand" href="#">
                    <img src="Logo.PNG" class="ui-icon-image" height="50" width="50" alt="" />
                </a>
            </div>

```
        <div id="navbar" class="collapse navbar-collapse navbar-responsive-collapse">
            <ul class="nav navbar-nav">
                <li><a href="Home.aspx">Home</a></li>
                <li class="active"><a href="Login.aspx">Login</a></li>
                <li><a href="AboutYou.aspx">About You</a></li>
            </ul>
        </div>

    </div>
</div>
    <!-- This page is meant to have the proper required validations and error message (in this case, label) and if the user enters their accessnet id and password, they are redirected to the home page.  There is not much else to this site….-->
     <div class="container" style="margin-top: 5%">
    <div class="col-md-4 col-md-offset-4">
        <div class="panel panel-default">
            <div class="panel-heading"><h1 class="panel-title"><strong>Log In </strong></h1></div>
            <div class="panel-body">
                </div>
                    <div class="form-group">
                        <div class="col-md-12">
                        <label for="exampleInputEmail1">Username</label>
                     <asp:TextBox runat="server" class="txt"  id="txtAccessNet" name="txtAccessNet" Text="" OnClick="btnLogin_Click" ></asp:TextBox>
                    </div>
                        </div>
                    <div class="form-group">
                        <div class="col-md-12">
                        <label for="exampleInputPassword1">Password</label>
                     <asp:TextBox runat="server" class="txt"  id="txtPassword" name="txtPassword" Text="" OnClick="btnLogin_Click" ></asp:TextBox>
                    </div>
                        </div>
                    <br />
                    <div style="margin-left: 80px">
                        <asp:Button runat="server" class="btn btn-md btn-success"  id="btnLogin" name="btnLogin" Text="Login" OnClick="btnLogin_Click" ></asp:Button>
    </div>
                    <div class="form-group">
                        <div class="col-md-12">
                    </div>
                        </div>
                    <asp:Label id="lblError" name="lblError" runat="server" style="position: relative" Text="* Your Username and Password are required fields!"></asp:Label>
                    <br />
                    <br />
                  <div class="form-group">
                <div class="col-md-12">
                    <br />
                </div>
            </div>

                    <div class="col-page col-sm-12 col-md-12">
            <div class="img-publicacion-home">
                <p>
                    <br />
                    <br />
                    <hr />
                    Footer:  
<a href="https://www.linkedin.com/in/pallavikotturu"><span class="label label-info">Author</span></a> 
<a href="https://cis.temple.edu/"><span class="label label-info">Temple University CIS</span></a> 
| <i class="icon-calendar"></i> November 8th, 2016 at 10:15 am
| <i class="icon-comment"></i> <a href="#">3 Comments</a>
| <i class="icon-share"></i> <a href="#">39 Shares</a>
```

</p>

```
                </div>
              </div>
                </form>
            </div>
        </div>
    </div>

         </form>
```

</body>
</html>
`
